addons/marker_flags/stringtable.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="ACE">
+    <Package name="Marker_Flags">
+        <Key ID="STR_ACE_Marker_Flags_DisplayName_Settings">
+            <English>Marker Flags</English>
+            <German>Markierungsfahnen</German>
+        </Key>
+        <Key ID="STR_ACE_Marker_Flags_PlaceAnywhere_DisplayName">
+            <English>Place Anywhere</English>
+            <German>Überall platzieren</German>
+        </Key>
+        <Key ID="STR_ACE_Marker_Flags_PlaceAnywhere_Description">
+            <English>Place marker flag anywhere regardless of surface.</English>
+        </Key>
+        <Key ID="STR_ACE_Marker_Flags_ActionPlace">
+            <English>Place Flag</English>
+            <German>Flagge platzieren</German>
+        </Key>
+        <Key ID="STR_ACE_Marker_Flags_ActionPickUp">
+            <English>Pick up</English>
```

Comment:
Would prefer title case for action names.

Reply:
```suggestion
            <English>Pick Up</English>
            <Chinese>提取點</Chinese>
            <French>Récupérer</French>
            <Spanish>Recoger</Spanish>
            <Italian>Recupera</Italian>
            <Polish>Podnieś</Polish>
            <Russian>Подобрать</Russian>
            <Czech>Vyzvednutí</Czech>
            <Portuguese>Pegar</Portuguese>
            <Korean>확보</Korean>
            <Chinesesimp>提取点</Chinesesimp>
            <Japanese>受け取り</Japanese>
            <Turkish>Al</Turkish>
```